packages/instant/src/components/ui/icon.tsx

```diff
@@ -1,17 +1,21 @@
 import * as React from 'react';
 
-import { ColorOption } from '../../style/theme';
-
 type svgRule = 'evenodd' | 'nonzero' | 'inherit';
 interface IconInfo {
     viewBox: string;
+    path: string;
     fillRule?: svgRule;
     clipRule?: svgRule;
-    path: string;
+    stroke?: string;
+    strokeOpacity?: number;
```

Comment:
I didn't anticipate all these other properties.  With these additions, I'm wondering if this Icon abstraction still feels good -- what do you think?

Reply:
I think we're good for now... worth talking about if it keeps on getting bigger though.